virt/virt-2-4-release-notes.adoc

```diff
@@ -4,6 +4,246 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-release-notes
 toc::[]
 
-Reminder: copy the 2.4 release notes into this branch/file to use as a reference for the 2.5 release notes.
+== About Red Hat {VirtProductName}
```

Comment:
We can remove this file from e-4.6
We only need a dummy file in master

Reply:
Removed `virt-2-4-release-notes.adoc`